.env.sample

```diff
@@ -1,3 +0,0 @@
-REACT_APP_CLIENT_ID=example
```

Comment:
the reason there is a check failing is because you deleted this file... please add this file back into your root folder